src/results/tests.ts

```diff
@@ -2661,41 +2661,1371 @@ export const valueTypeSpecializationTests: Test[] = [
     skip: true,
   },
   {
-    name: 'Int',
+    name: 'String',
+    query: `
+      value type MyType = Int, String
+      def v = ^MyType[1, "abc"]
+      def output = #(v)
+    `,
+    typeDefs: [
+      {
+        type: 'ValueType',
+        typeDefs: [
+          {
+            type: 'Constant',
```

Comment:
There's a bug on the server side. This constant in the metadata is not being returned currently. We ignore all these tests at the moment.